src/routes/index.ts

```diff
@@ -22,5 +23,6 @@ router.use('/plans', trainingPlan);
 router.use('/categories', category);
 router.use('/machines', machine);
 router.use('/exercises', exercise);
+router.use('/planExercise', planExercise);
```

Comment:
i dont want a separate route for plan exercises.
i want the plan exercises to be under the route `/api/plans/:plan_id/exercises` just like the user plans